node_modules/col-assets/tests/test-assets.js

```diff
@@ -15,6 +15,7 @@
 
```

Comment:
How do we want to deal with the tests? Are we assuming a running Canvas instance or should we mock up responses? If the former, do we know if we'll be able to make that work on Travis?